source/tests/system/nifgen_driver_api_tests.cpp

```diff
@@ -1,5 +1,8 @@
 #include <gtest/gtest.h>
 
+#define _USE_MATH_DEFINES
```

Comment:
Note: `#define _USE_MATH_DEFINE` statement is needed to use the `M_PI` constant in the helper method below.

Reply:
Removed the usage of sin and M_PI